logs.rst

```diff
@@ -69,3 +69,19 @@ Results for Pruning Algorithms -
 |            |         +---------------+----------------+-----------------+----------+
 |            |         |      3/3      |      18.99     |        5        |  98.90 % |
 +------------+---------+---------------+----------------+-----------------+----------+
+
+Results for  Quantization - 
+===================================
+
++------------+---------+---------------+----------------+-----------------+----------+--------------+
+| Model Used | Dataset |    Model      |    Size (MB)   | Training epochs | Accuracy | Time elapsed |
++============+=========+===============+================+=================+==========+==============+
+|  ResNet152 |  MNIST  |     Base      |       4.15     |        1        |  0.9582  |  28.84 s     |
+|            |         +---------------+----------------+                 +----------+--------------+
+|            |         |   Quantized   |       0.56     |                 |  0.9583  |  28.71 s     |
++            +---------+---------------+----------------+-----------------+----------+--------------+
+|            | CIFAR10 |     Base      |       4.15     |        1        |  0.9582  |  28.84 s     |
+|            |         +---------------+----------------+                 +----------+--------------+
+|            |         |   Quantized   |       0.56     |                 |  0.9583  |  28.71 s     |
++------------+---------+---------------+----------------+-----------------+----------+--------------+
```

Comment:
One more thing here it will be good to mention which method did we use for quantization.

Reply:
How are both MNIST and CIFAR10 getting the same metrics :P

Reply:
They are not lol. I just copy pasted to enter the result later but forgot